custom/logistics/tasks.py

```diff
@@ -66,9 +73,7 @@ def stock_data_task(domain, endpoint, apis, test_facilities=None):
         # todo: see if we can avoid modifying the list of facilities in place
         if idx == 0:
             facilities = facilities_copy
-    save_stock_data_checkpoint(checkpoint, 'product_stock', 100, 0, start_date, None, False)
-    checkpoint.start_date = None
-    checkpoint.save()
+    save_stock_data_checkpoint(checkpoint, 'product_stock', 100, 0, None, None)
```

Comment:
sorry about that. https://github.com/dimagi/commcare-hq/pull/5919

got confused between `date` and `start_date` in my cleanup